presets/ragengine/main.py

```diff
@@ -32,6 +45,18 @@
 # Initialize RAG operations
 rag_ops = VectorStoreManager(vector_store_handler)
 
+@app.get("/metrics", tags=["Monitoring"])
+async def metrics():
+    """
+    Expose Prometheus metrics.
+    Returns metrics data in Prometheus exposition format.
+    """
+    try:
+        return Response(generate_latest(), media_type=CONTENT_TYPE_LATEST)
```

Comment:
mind putting brief comment here on what `generate_latest` and `content_type_latest` are doing or linking to documentation